pages/views/frontend.py

```diff
@@ -56,63 +56,32 @@ def frontend(request):
         request.session.set_expiry(0)
         
         # grab client cert form the request create user hash, ignore if no cert info is found in request
-        
         try:
             certInfo = getCert(request)
+            cftsUser = getOrCreateUser(request, certInfo)
+
+            if cftsUser == None or not request.user.is_authenticated:
```

Comment:
- remove user.is_authenticated from line 63, this would force a valid PKI user who returned a user record to sign in even though it wouldn't be required for them to submit a request

Reply:
Done